server/src/main/java/org/apache/iotdb/db/qp/physical/sys/ShowPlan.java

```diff
@@ -44,6 +65,34 @@ public ShowContentType getShowContentType() {
     return showContentType;
   }
 
+  public PartialPath getPath() {
+    return this.path;
+  }
+
+  public int getLimit() {
+    return limit;
+  }
+
+  public void setLimit(int limit) {
+    this.limit = limit;
+  }
+
+  public int getOffset() {
+    return offset;
+  }
+
+  public void setOffset(int offset) {
+    this.offset = offset;
+  }
+
+  public boolean hasLimit() {
+    return hasLimit;
+  }
```

Comment:
The sonar test shows these lines are duplicated with some lines in `ShowTimeseriesPlan`. How do you think remove the lines in `ShowTimeseriesPlan`?

Reply:
code smell solved. thank you :)